DiscordBot/Commands/ICommandGetter.cs

```diff
@@ -6,6 +6,6 @@ namespace DiscordBot.Commands
 {
     interface ICommandGetter
```

Comment:
I thought of a better name for the class: `CommandProvider` :)

Reply:
wow!